configs/components/runtime-bolt.rb

```diff
@@ -19,7 +19,7 @@
 
   else # Linux and Solaris systems
     libbase = platform.architecture =~ /64/ ? 'lib64' : 'lib'
-    libdir = "/opt/pl-build-tools/#{libbase}"
+    libdir = platform.name =~ /fedora-29/ ? "/usr/lib64" : "/opt/pl-build-tools/#{libbase}"
```

Comment:
Oh gosh, actually, I just remembered -- would this work if you added fedora-29 to the conditional just above (https://github.com/puppetlabs/puppet-runtime/pull/133/files#diff-8da58944b3f541d50ae6f6257ad40b8cL16) instead? SLES 15 is another platform that doesn't use pl-build-tools, since its compiler is new enough (also, if that does work, adding a more descriptive comment than is currently there would surely help)

Reply:
Thanks for the review. I will try an ad-hoc build with your suggestion. I agree that is less than helpful.